tests/requirements/py3.txt

```diff
@@ -7,7 +7,7 @@ docutils >= 0.19
 geoip2 >= 4.8.0
 jinja2 >= 2.11.0
 numpy >= 1.26.0
-Pillow >= 10.1.0; sys.platform != 'win32' or python_version < '3.14'
```

Comment:
Is this working fine with `win32` now? I couldn't catch why that condition was removed.

Reply:
I think the restriction was added defensively during a very early phase of testing against 3.14 alphas, and wheels for windows may not have been available yet. (48fa531) Removing it if possible was always the plan.